code/mob/living/life/disability.dm

```diff
@@ -7,9 +7,9 @@
 
 		// moved drowsy, confusion and such from handle_chemicals because it seems better here
 		if (owner.drowsyness)
-			owner.drowsyness--
-			owner.change_eye_blurry(2)
-			if (prob(5))
+			owner.drowsyness = max(0, owner.drowsyness - mult)
```

Comment:
```suggestion
			owner.drowsyness = max(0, owner.drowsyness - 1*mult)
```

Reply:
More consistent with how its applied in other places (e.g.: all over reagent code)